dask/array/reshape.py

```diff
@@ -226,6 +231,36 @@ def reshape(x, shape, merge_chunks=True):
         x = x.rechunk({i: 1 for i in range(din - dout)})
 
     inchunks, outchunks = reshape_rechunk(x.shape, shape, x.chunks)
+    # Check output chunks are not too large
+    max_chunksize_in_bytes = np.prod([max(i) for i in outchunks]) * x.dtype.itemsize
+    if (
+        limit is None
+        and config.config["array"]["slicing"]["split-large-chunks"] is not False
```

Comment:
```suggestion
        and config.get("array.slicing.split-large-chunks") is not False
```

I think this is more readable and similar to the next line.